src/gwt/src/org/rstudio/studio/client/server/remote/RemoteServer.java

```diff
@@ -1390,6 +1391,12 @@ public void createShinyApp(String appName,
       sendRequest(RPC_SCOPE, CREATE_SHINY_APP, params, requestCallback);
    }
    
+   public void getActiveDocumentContextCompleted(JsObject context,
```

Comment:
We typically create strongly typed JavaScriptObject subclasses for this sort of thing (just to ensure that the contract is clearly documented and enforced)